c7n/resources/rds.py

```diff
@@ -952,6 +952,18 @@ def process_snapshot_set(self, snapshots_set):
                 DBSnapshotIdentifier=s['DBSnapshotIdentifier'])
 
 
+@actions.register('modify-security-groups')
+class RDSModifyGroups(ModifyGroupsAction):
```

Comment:
modify security groups in class name, dbsubnet groups are the other consideration, just disambiguating.